tracer/test/snapshots/HangfireTests.verified.txt

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: Hangfire.onPerform,
+    Resource: JOB TestJob.Execute,
+    Service: Samples.Hangfire-Hangfire,
```

Comment:
If this is an internal span, I think that we should keep the same service name, instead of doing `{service}-hangfire`